assets/components/lvms/topolvm-node_daemonset.yaml

```diff
@@ -78,16 +60,16 @@ spec:
           protocol: TCP
         resources:
           requests:
-            cpu: 1m
-            memory: 16Mi
+            cpu: 2m
+            memory: 35Mi
```

Comment:
Those are still small numbers, but they're doubling. Is that absolutely necessary?

Reply:
I am removing an entire container whose work is now done in this one. Total usage is still down, but we can of course always reduce further if the requests are continously lower than 70% requests

Reply:
Nice, that sounds like a good improvement overall.